content/guides/template-gds/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 'GDS File Creation from Template'
+linkTitle: 'Create GDS Files With Our Github Template'
+description: 'The detailed procedure to get GDS from a wokwi template repo'
+weight: 60
+---
+
+
+This guide will help you through the essential steps to set up a GitHub repository, enable actions, and successfully commit your design for TinyTapeout. Follow each step carefully.
+
+## Use the Template Repository
+-------------------------------
```

Comment:
remove the horizontal rules - try to follow the current (undocumented) style of the existing docs